internal/cli/usercmd/client.go

```diff
@@ -476,17 +480,27 @@ func (c *EpinioClient) ServiceDetails(name string) error {
 	}
 	serviceDetails := resp.Details
 
-	msg := c.ui.Success().WithTable("", "")
+	msg := c.ui.Success().WithTable("", "", "Access Path")
 	keys := make([]string, 0, len(serviceDetails))
 	for k := range serviceDetails {
 		keys = append(keys, k)
 	}
 	sort.Strings(keys)
 	for _, k := range keys {
-		msg = msg.WithTableRow(k, serviceDetails[k])
+		path := ""
+		switch k {
+		case "Status":
```

Comment:
I think "Status" is no more?

Reply:
Right, with custom as service this is gone. Will fix.

Reply:
Done, commit 37c434ba7258d68c130d1b7bba04eecc3b17ed3d